5calls/app/src/main/java/org/a5calls/android/a5calls/adapter/IssuesAdapter.java

```diff
@@ -81,6 +82,11 @@ public void setAllIssues(List<Issue> issues, int errorType) {
         }
     }
 
+    public void setStarredIssues(List<String> issueIds) {
+        starredIssueIds = issueIds;
```

Comment:
it might be easier to annotate `mIssues` with `isStarred` rather than having a second list of `starredIssueIds`? Or have a starredIssueIds be a `Set<String> mStarredIssues` for easy lookups.